lib/reader.js

```diff
@@ -23,12 +23,18 @@ function Reader(config) {
   }
   assert(config.queueUrl, 'queueUrl required');
 
+  if (config.waitTimeSeconds) {
+    assert(config.waitTimeSeconds >= 0 && config.waitTimeSeconds <= 20,
```

Comment:
I wonder if it might be best to leave this validation to AWS, in case they ever change... 🤔

Reply:
true, lets do that.